### ISSUE TITLE
Padding left by default is not working - possible fix found?

### ISSUE THREAD

**Issue by pedrorfernandes** [masayuki0812/c3#999]
I've updated C3 from 0.3.0 to 0.4.9 and noticed the padding left is not being calculated at all

As an example: the Y axis only shows '00'
http://jsfiddle.net/masayuki0812/7kYJu/
Screenshot: 
![screenshot 2015-02-13 16 53 29](https://cloud.githubusercontent.com/assets/1915671/6191477/a18dba44-b3a1-11e4-8022-2f8df92c1581.png)

I've traced down the error to line 4372 of c3.js:
https://github.com/masayuki0812/c3/blob/35e5a63de53cd7c336a49612cb42b143098dccef/c3.js#L4372

Inside the function c3_chart_internal_fn.getMaxTickWidth
```javascript
if (box.left >= 0 && maxWidth < box.width) { maxWidth = box.width; }
```
In my case, box.left is always negative, maxWidth will always remain 0, and thus getMaxTickWidth() will always return 0.

Removing ```box.left >= 0``` seems to resolve the issue

**Comment by ttamminen** [comment]
https://github.com/masayuki0812/c3/issues/994

**Comment by masayuki0812** [comment]
Thanks for looking into this. I think this was fixed in #994 , so please try `0.4.10-rc4`. I'll release `0.4.10` shortly.

**Comment by masayuki0812** [comment]
`0.4.10` has been released, so let me close at the moment. Thanks.
